pkg/kubectl/cmd/drain.go

```diff
@@ -112,12 +112,12 @@ func NewCmdCordon(f cmdutil.Factory, out io.Writer) *cobra.Command {
 }
 
 var (
-	uncordon_long = templates.LongDesc(`
-		Mark node as schedulable.`)
+	uncordon_long = templates.LongDesc(i18n.T(`
+		Mark node as unschedulable.`))
```

Comment:
Additionally cordon/unschedulable and uncordon/schedulable is messed up as well.